client.go

```diff
@@ -312,7 +312,13 @@ func (c *dedicatedSingleClient) release() {
 }
 
 func (c *singleClient) isRetryable(err error, ctx context.Context) bool {
-	return err != nil && err != ErrDoCacheAborted && atomic.LoadUint32(&c.stop) == 0 && ctx.Err() == nil
+	if err == nil || err == ErrDoCacheAborted || atomic.LoadUint32(&c.stop) != 0 || ctx.Err() != nil {
+		return false
+	}
+	if redisErr, ok := err.(*RedisError); ok {
+		return redisErr.IsLoading()
+	}
+	return true
 }
 
 func isRetryable(err error, w wire, ctx context.Context) bool {
```

Comment:
It turns out this `isRetryable` used by `dedicatedSingleClient` should also be updated.